src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityThread.test.js

```diff
@@ -1,22 +1,68 @@
 // @flow
 import React from 'react';
-import { shallow, ShallowWrapper } from 'enzyme';
-
+import { render, screen, fireEvent } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
 import ActivityThread from '../ActivityThread.js';
+import localize from '../../../../../../test/support/i18n';
+import { replies } from '../fixtures';
+import messages from '../messages';
 
+jest.mock('react-intl', () => jest.requireActual('react-intl'));
```

Comment:
Is this needed?

Reply:
It is needed to prevent mocking react-intl components from `/__mocks__/react-intl.js`